### DISCUSSION TITLE
Help!! selecting a text/string using WDIO

### DISCUSSION THREAD

**Discussion by athul-snair** [webdriverio/webdriverio#7775] [Q&A]
Hey Guys,
Is there a way to select a certain text in a given page?By select I meant the action of doing a mouse down ,drag and mouse up.
for example:
![Screenshot 2021-12-03 at 11 37 54 AM](https://user-images.githubusercontent.com/91887687/144555648-44665a7f-3c80-4b2a-a6f5-c7f538fc8610.png)
![Screenshot 2021-12-03 at 11 38 10 AM](https://user-images.githubusercontent.com/91887687/144555660-f1ff34c7-369f-4790-aa63-0417f9dfe143.png)
Any help would be appreciated.

**Comment by erwinheitzman** [comment]
As far as I'm aware there no other way then to use javascript for this. This can be done using the native text selection functions and the `execute` command.

Something like this should work: https://stackoverflow.com/questions/6139107/programmatically-select-text-in-a-contenteditable-html-element
